### ISSUE TITLE
Cocoapods crashes on install

### ISSUE THREAD

**Issue by willedanielsson** [react-native-community/lottie-react-native#566]
### Description
Cocoapods crashes while following the installation steps with
```
[!] CocoaPods could not find compatible versions for pod "lottie-ios":
In Podfile:
    lottie-ios (from `../node_modules/lottie-ios`)

    lottie-react-native (from `../node_modules/lottie-react-native`) was resolved
to 3.1.1, which depends on lottie-ios (~> 3.0.3)
```

### Steps to Reproduce

1. `yarn add lottie-react-native`
2. `yarn add lottie-ios@3.1.3`
3. `cd ios && pod install`

**Expected behavior:** [What you expected to happen]
Pod installation to work

**Actual behavior:** [What actually happened]
It crashes with above error
### Versions

    "react-native": "^0.60.4",
    "lottie-react-native": "^3.1.1",
    "lottie-ios": "3.1.3",

However, if one does not _yarn adds_ the lottie-ios, the _pod install_ will work and point to lottie-ios (3.0.7)

**Comment by hoangtranminhtai** [comment]
I have the same issue

**Comment by hoangtranminhtai** [comment]
@willedanielsson You can change git repo in package.json:
    "lottie-ios": "^3.1.3",
    "lottie-react-native": "git+https://github.com/react-native-community/lottie-react-native.git",
It won't crashes with above error
If you have issues with build your iOS project. Follow this step:
https://stackoverflow.com/questions/52536380/why-linker-link-static-libraries-with-errors-ios
Goodluck

**Comment by willedanielsson** [comment]
> @willedanielsson You can change git repo in package.json:
> "lottie-ios": "^3.1.3",
> "lottie-react-native": "git+https://github.com/react-native-community/lottie-react-native.git",
> It won't crashes with above error
> If you have issues with build your iOS project. Follow this step:
> https://stackoverflow.com/questions/52536380/why-linker-link-static-libraries-with-errors-ios
> Goodluck

Yeah this solution works for now for users who want to get started. 
I get the error `ld: library not found for -lDoubleConversion` but that seems like another issue than this one 👍

**Comment by emilioicai** [comment]
this is fixed in 3.2.1, please give it a try and let me know if this is still an issue. I will close this for the time being.

**Comment by anniewey** [comment]
@emilioicai I couldn't update to `3.2.1` due to #543 issue in Android
